app/bridge/server.js

```diff
@@ -46,14 +56,42 @@ function createServer(name, version, methods) {
       interface: this.getContract()
     });
 
+    // we keep a ref to the listener to be able to remove it.
+    channel.onMessageListener = e => {this.onmessage.call(this, channel, e.data);};
```

Comment:
Maybe. I tried everything and still have the leak. When I find the root cause, I'll try to deactivate that.